### ISSUE TITLE
Accessibility improvements

### ISSUE THREAD

**Issue by saracarl** [benwbrum/fromthepage#2356]
We have a handful of places that are missing a11y details:

- docset listing on owner page is missing a <ul> around the <li>s
- thumbnail images for collections need alt text
- search fields need labels

Details here:  https://drive.google.com/drive/folders/10-mFluGhEqJvaRk9bYyhZnMu6jDgkzzL
